modules/ROOT/partials/integrations/jquery-quick-start.adoc

```diff
@@ -0,0 +1,197 @@
+The https://github.com/tinymce/tinymce-jquery[Official {productname} jQuery component] integrates {productname} into jQuery projects.
+
+
+NOTE: We are pleased to provide integrations/code guidance to help you build great products with {productname}. If you have queries about this integration, post your question on Stack Overflow using the link:{communitysupporturl}[`{prodnamecode}` tag].
+
+== Procedure
+This procedure creates a basic jQuery integration containing a {productname} editor based on our xref:basic-example.adoc[Basic example].
+
+. Open a HTML file.
+
+// ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~PACKAGE INSTALL~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+ifeval::["{productSource}" == "package-manager"]
+. Install jQuery, {productname}, the {productname} jQuery integration and the library `fs-extra` from NPM:
++
+[source,shell]
+----
+npm install --save jquery tinymce @tinymce/tinymce-jquery fs-extra
+----
+
+. Setup a `postinstall` script to copy the packages into a public directory:
++
+.postinstall.js
+[source,js]
+----
+const fse = require('fs-extra');
+const path = require('path');
+
+const nodeModulesDir = path.join(__dirname, 'node_modules');
+const publicDir = path.join(__dirname, 'public');
+
+fse.copySync(path.join(nodeModulesDir, 'jquery'), path.join(publicDir, 'jquery'), { overwrite: true });
+fse.copySync(path.join(nodeModulesDir, 'tinymce'), path.join(publicDir, 'tinymce'), { overwrite: true });
+fse.copySync(path.join(nodeModulesDir, '@tinymce', 'tinymce-jquery'), path.join(publicDir, 'tinymce-jquery'), { overwrite: true });
+----
++
+.package.json
+[source,json]
+----
+{
+  // ... snip ...
+  "scripts": {
+    "postinstall": "node ./postinstall.js"
+  }
+}
+----
+
+. Assuming the `public` directory is served at `+/public+`, add source scripts, such as:
++
+[source,html]
+----
+<script src="/public/jquery/jquery.min.js"></script>
+<script src="/public/tinymce/tinymce.min.js"></script>
+<script src="/public/tinymce-jquery/tinymce-jquery.min.js"></script>
+----
++
+For information on self-hosting {productname}, see: link:advanced-install.html[Installing {productname}].
```

Comment:
I don't know what to replace this link with? Any suggestions?

- [ ] Replace link

Reply:
That's broken elsewhere as well... we have DOC-1586 logged to solve what to do with those old index pages so this is probably fine to leave it as is for now.

Reply:
Ok, I'll leave it as is.